app/javascript/bundles/Container/components/lessons/DiffusionScript.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+const DiffusionScript = () => {
+  let width = 1400,
```

Comment:
If you change the width to 500 here, the view will look a bit better on mobile and narrow devices

Reply:
I changed the width to 1400 to get the animation to be at the center of the page.  But I see your point now that I'm looking at the responsive browser.  I'm not too familiar with how to make React components responsive.  However, I'll change it back to the original width and see if I can use css to center it instead. Thanks!

Reply:
I reverted back to the original width of 960. I tried several ways with css but could not change the React component's spacing.  The animation is center-screen for mobile devices, but is still positioned more to the left on laptop screens.

Reply:
Great, I think 960 looks good!